README.md

```diff
@@ -126,8 +126,6 @@ Existing messages in the queue will be lost so please be aware of this.
 % mix task_bunny.queue.reset
 ```
 
-You need to redefine a queue when you want to change the retry interval for a queue.
```

Comment:
This is no longer a valid statement, the expiration is now per message not per queue which was the case when this comment was added.